Packs/FeedRSS/pack_metadata.json

```diff
@@ -8,11 +8,9 @@
     "email": "",
     "created": "2020-10-21T11:42:09Z",
     "categories": [
-        "Data Enrichment & Threat Intelligence"
-    ],
-    "tags": [
-        "TIM"
```

Comment:
Bar & I are currently discussing the meaning of "Threat Intelligence Management."
We need to decide if this will fit only integrations which enable IoC & Feed management or not.
We'll probably do a v2 revamp shortly.